source/_docs/email.md

```diff
@@ -138,11 +138,20 @@ We strongly recommend that you do not use the local MTA (postfix) as described [
 No, mail logs are not available for download and we do not recommend using the local MTA (postfix).
 
 ### What ports are recommended by Pantheon?
-Configuring mail to use port 25, 465 or 587 is strongly discouraged because they attract SPAM activities. Make sure that your service provider allows traffic on a port other than those mentioned and that you have correctly configured your site to use that port.
+Pantheon strongly encourages using ports other than `25`, `465` or `587` to send email because those ports are often blocked by service providers as an anti-spam measure.  Make sure that your service provider allows traffic on a port other than those mentioned and that you have correctly configured your site to use that port.
 
-### Can Pantheon provide, publish, or support SPF records?
+### Are there SPF records for Pantheon's local MTA (postfix)?
+If you are using Pantheon's local MTA ([not recommended](#outgoing-email)), and your domain contains an SPF record, then you should include Pantheon's SPF record, as shown below:
 
-As consumers of cloud infrastructure, we don’t have control over our IP ranges and they are subject to change without our notice. Publishing an SPF record would imply assurance on our end that it can work, which would be very difficult to guarantee given these circumstances. We take the decision of what we support and what we don’t very seriously, and at this time we’re not in a position to support SPF records.
+```
+v=spf1 include:spf.example.com include:spf.pantheon.io -all
```

Comment:
@garydylina suggested that most folks will probably use `~all`, so perhaps we should use that here as well.

Reply:
I didn't realize that `-all` and `~all` were different, will adjust.